python/lsst/summit/utils/astrometry/plotting.py

```diff
@@ -143,3 +143,4 @@ def plot(exp, icSrc=None, filteredSources=None, saveAs=None,
 
     plt.gcf().clear()
     del fig
+    del data
```

Comment:
If this still isn't working, you might want to try `gc.collect()`.  If that doesn't work, @natelust is the the expert.